config/config_embed.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	_ "embed"
+)
+
+// Embedding configuration files
```

Comment:
Either make a multi-var declaration like:
```
    var (
        ...
    )
```
or add a corresponding comment to every variable.